tdrs-backend/tdpservice/reports/views.py

```diff
@@ -22,6 +29,16 @@ class ReportFileViewSet(ModelViewSet):
     serializer_class = ReportFileSerializer
     queryset = ReportFile.objects.all()
 
+    @action(methods=["get"], detail=True)
+    def download(self, request, pk=None):
+        """Retrieve a file from s3 then stream it to the client."""
+        record = DownloadReportFileSerializer(ReportFile.objects.get(id=pk))
+
+        response = StreamingHttpResponse(streaming_content=record.data['file'])
```

Comment:
```suggestion
        response = StreamingHttpResponse(
            FileWrapper(record.file),
            content_type='txt/plain'
        )
```
You'll need to also import `from wsgiref.util import FileWrapper` for this one, but I found that the files were coming back as binary without these changes.